src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header 
+      class="header"
+    >
+      <img 
+        src="./images/logo.png" 
+        alt="logo"
+        class="logo"
+      >
```

Comment:
```suggestion
<a href="#" class="logo">
   <img 
        src="./images/logo.png" 
        alt="Moyo logo"
        class="logo__image"
    >
</a>
      
```

Reply:
Not fixed previous comment 
![image](https://user-images.githubusercontent.com/70009205/209987815-ab588d01-468e-4851-9a07-1e0d1d9b70c1.png)
Please fix it